### DISCUSSION TITLE
Installation stalling at running initial configuration

### DISCUSSION THREAD

**Discussion by negus6** [Security-Onion-Solutions/securityonion#2890] [Unsupported Versions] [answered]
Trying to install manager node on ESXI (16 GB Ram, 6 procs)but the install keeps hanging at running initial configuration steps. The install logs show errors that the system is attempting to resolve a master hostname of 'salt', but that not the hostname of the system.![image](https://user-images.githubusercontent.com/44405268/107169846-e26c0500-698c-11eb-889a-b9f179716ab4.png)![image](https://user-images.githubusercontent.com/44405268/107169851-e730b900-698c-11eb-8ca0-fb14507eb53d.png)

**Comment by iriwsky** [comment]
I have the same issue at the installation.  Also looking for solution.

**Comment by defensivedepth** [comment]
Is this a network install? Can you confirm that you are running the setup with `sudo` ?

Also, any errors in `/root/sosetup`?

**Reply by iriwsky** [reply]
Thank you for the response.  For mine, I am using an iso (ver 2.3.21).  I am using vmware workstation 16 pro, centos 7.  The log just contains that I cancelled the setup.

**Reply by defensivedepth** [reply]
Did you run setup manually?

Can you post the entire log, sanitized as needed?

**Reply by iriwsky** [reply]
I am going to class; I will upload as soon as I get off class in an hour.  Thank you again!

**Comment by iriwsky** [comment]
I have not forgot; just a total student Linux noob.  I'm getting 'permission denied' to root/sosetup.  I will post the file when I figure out access to the log.

**Reply by defensivedepth** [reply]
you will need to use `sudo`

**Comment by negus6** [comment, marked as answer]
Here's the full log. I ran the setup manually using the install script in /root/SecurityOnion/setup (using iso the command argument)

[sosetup.log](https://github.com/Security-Onion-Solutions/securityonion/files/5966333/sosetup.log)

Why is the setup trying to resolve 'salt' as "master"? Wouldn't the SO-Manager note itself be the "master"? Also, there was nothing in the setup where the name 'salt' was entered as a parameter. Do I need to modify something in the host file or manually in the setup script?

**Reply by defensivedepth** [reply]
Ok this is a confirmed bug - you should be able to reinstall and not have problems, but this bug is preventing that. The best thing to do right now is start with a fresh VM and reinstall the ISO.

I have created an issue to track this bug - it will be fixed in our next release.

https://github.com/Security-Onion-Solutions/securityonion/issues/2936

**Reply by negus6** [reply]
Awesome... this worked. Thanks!

**Comment by iriwsky** [comment]
Thank you!  Will reinstall with a fresh VM.

**Comment by iriwsky** [comment]
Re-installation using a new vm worked!  Closing the discussion for me.  Thank you!

**Reply by defensivedepth** [reply]
Great to hear!

**Reply by ssi0202** [reply]
hi i still see this issue with 2.3.70 i have tried with a clean install and the network card in bridged mode to ensure DNS conectivity

**Reply by dougburks** [reply]
@ssi0202 Per https://github.com/Security-Onion-Solutions/securityonion/discussions/1720, please start a new discussion rather than replying to somebody else's discussion. Thanks!
